src/estimators/ordinary_kriging.jl

```diff
@@ -30,7 +30,7 @@ function set_constraints_lhs!(estimator::OrdinaryKriging, LHS::AbstractMatrix, X
   nothing
 end
 
-factorize(estimator::OrdinaryKriging, LHS::AbstractMatrix) = lu(LHS, check=false)
+factorize(estimator::OrdinaryKriging, LHS::AbstractMatrix) = bunchkaufman(Symmetric(LHS), check=false)
```

Comment:
Nice, did you notice improvements compared to LU factorization? Should we also use this factorization in other variants of Kriging that also have symmetric matrices?

Reply:
Yes, but only with large datasets and > 50 neighbours.
All kriging systems are symmetric. For SO we already have the cholesky factorisation, now for OK the bunchkaufman factorisation. Let me check others. I assume that when building the linear system we are not calculating the "second" have.

Reply:
Also worth checking the list of available factorizations in the latest versions of the language :+1:

This is what I get in Julia v1.5:

```julia
julia> using LinearAlgebra

julia> subtypes(Factorization)
19-element Array{Any,1}:
 BunchKaufman
 Cholesky
 CholeskyPivoted
 Eigen
 GeneralizedEigen
 GeneralizedSVD
 GeneralizedSchur
 Hessenberg
 LDLt
 LQ
 LU
 LinearAlgebra.QRCompactWY
 QR
 QRPivoted
 SVD
 Schur
 SuiteSparse.CHOLMOD.Factor
 SuiteSparse.SPQR.QRSparse
 SuiteSparse.UMFPACK.UmfpackLU
```

Reply:
I already did its for this PR.